test/browser/google-blog-feed-loader.test.js

```diff
@@ -0,0 +1,114 @@
+var should = require('should');
+
+var loader = require('../../lib/google-blog-feed-loader');
+
+var RAW_FEED = {
+  "feedUrl": "https://blog.webmaker.org/tag/teachtheweb/feed",
+  "title": "Mozilla Learning » #teachtheweb",
+  "link": "https://blog.webmaker.org",
+  "author": "",
+  "description": "People, products and programs to teach, learn and make the Web",
+  "type": "rss20",
+  "entries": [
+    {
+      "title": "What’s next for Thimble?",
+      "link": "https://blog.webmaker.org/whats-next-for-thimble",
+      "author": "Hannah Kane",
+      "publishedDate": "Tue, 12 May 2015 10:47:33 -0700",
```

Comment:
Just a note:

I'm not sure if it's the [API doc](https://developers.google.com/feed/v1/devguide#resultJson) being out of date - it doesn't include the "day of week" bits (see `publishedDate` section).  However when I ran Google API it did return date info in this format "Tue, 12 May 2015 10:47:33 -0700".